test/integration/controller/jobs/job/job_controller_test.go

```diff
@@ -790,6 +1014,163 @@ var _ = ginkgo.Describe("Job controller interacting with scheduler", ginkgo.Orde
 		util.ExpectResourceFlavorToBeDeleted(ctx, k8sClient, spotUntaintedFlavor, true)
 	})
 
+	ginkgo.When("the queue has admission checks", func() {
```

Comment:
why two tests?

Reply:
This one integrates with scheduler, I thought it might be valuable, but I'm also ok dropping it.

Reply:
it should be ok to drop. We might be better of doing an E2E using ProvReq.

Reply:
Dropped, but actually I found it flakes https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/kubernetes-sigs_kueue/1180/pull-kueue-test-integration-main/1712147680169299968, as the workload doesn't get admitted. It passes locally for me most of the time. I would like to investigate as a follow up. It seems unrelated to the changes.

Reply:
In case of failures, I get "Workload is inadmissible because ClusterQueue is inactive". After adding `util.SetAdmissionCheckActive(ctx, k8sClient, admissionCheck, metav1.ConditionTrue)` it seems to pass consistently. Anyway, dropping the test. Just updated the other one.

Reply:
That makes sense. I guess that this failure happened due to https://github.com/kubernetes-sigs/kueue/pull/1193 being merged into the main even though this PR opened first.